sense/finetuning.py

```diff
@@ -97,7 +97,7 @@ def __getitem__(self, idx):
         return [features, self.labels[idx], temporal_annotation]
 
 
-def generate_data_loader(features_dir, tags_dir, label_names, label2int,
+def generate_data_loader(project_config, features_dir, tags_dir, label_names, label2int,
```

Comment:
Side note: The proper way to attribute these changes to @guillaumebrg and keep version history clean would have been to `cherry-pick` his commit. Did you try that?

Reply:
I specifically did not `cherry-pick` it, I used `checkout` to get the files from his branch.